spec/functional/command/outdated_spec.rb

```diff
@@ -39,6 +38,65 @@ module Pod
       UI.output.should.not.include('UIKit')
     end
 
+    it 'tells the user about outdated pods that can be updated in green' do
+      pod_name = 'BlocksKit'
+
+      current_version_string = mock
+      current_version_string.expects(:green).returns('1.0').once
+      current_version = mock
```

Comment:
this seems redundant? are we recreating the mock?

Reply:
sorry, too early in the morning, two different variables. Carry on.

Reply:
👍